Dockerfile

```diff
@@ -15,11 +15,11 @@ RUN git clone https://github.com/aerogear/aerogear.org aerogear
 WORKDIR /home/aerogear/aerogear
 
 ## Setup Project
-  RUN  /bin/bash -l -c "rvm install ruby 2.4.1"
- RUN  /bin/bash -l -c "rvm gemset create aerogear"
- RUN  /bin/bash -l -c "rvm use ruby-2.4.1@aerogear"
- RUN  /bin/bash -l -c "gem install bundler"
- RUN  /bin/bash -l -c "bundle install --path vendor"
+RUN  /bin/bash -l -c "rvm install ruby 2.4.1"
```

Comment:
each one of these results in a layer might be worth grouping them together?

Reply:
agreed - found it this way, but I'm happy to refactor as a separate commit for quicker builds. let me add that in.

Reply:
The build locally took >5mins.
I think there's 2 reasons for that:
* cloning the repo into the docker container
* building ruby (and installing its deps)

To shorten that time:
* Could we push a base layer with all the ruby stuff already in it?
* Could we mount the src from the host into the running container instead of cloning it again?